docs/gen-ai/gen-ai-agent-spans.md

```diff
@@ -331,6 +332,16 @@ system instructions.
 See [Recording content on attributes](/docs/gen-ai/gen-ai-spans.md#recording-content-on-attributes)
 section for more details.
 
+**[15] `gen_ai.tool_definitions`:** The value of this attribute matches source system tool definition format.
```

Comment:
Why is it `gen_ai.tool_definitions` and not `gen_ai.tool.definitions`? There are 6 attributes starting with `gen_ai.tool.` including 2 that you're adding here, so this inconsistency seems confusing.

Reply:
@alexmojaki : good catch, please allow me to make it consistent.

Reply:
@alexmojaki : fixed and pushed the changes. Please review..
Resolving the comment meanwhile.